openprescribing/frontend/management/commands/delete_measure.py

```diff
@@ -11,19 +11,19 @@ class Command(BaseCommand):
     def handle(self, measure_id, **options):
         if not options["delete_live_measure"]:
             if not measure_id.startswith(settings.MEASURE_PREVIEW_PREFIX):
-                # We want these errors to be visble to users who run via ebmbot, but
-                # ebmbot doesn't show stderr so we can't just raise CommandError here
+                # We want these errors to be visble to users who run via ebmbot but the
+                # only way to achieve that is to write them to stderr and exit 0 :(
```

Comment:
nit: comment doesn't match reality.

I think the comment (and the the one below) should `s/stderr/stdout